Update aliases.mdx (#6986)
withastro/docs: src/content/docs/en/guides/aliases.mdx

> :::note
> Make sure `compilerOptions.baseUrl` is set so the aliased paths can be resolved.
> You may need to restart your development server if it's running.
> :::
> 
> With this change, you can now import using the aliases anywhere in your project:

Reviewer: ```suggestion
:::note
Make sure `compilerOptions.baseUrl` is set so the aliased paths can be resolved.
:::

The development server will automatically restart after this configuration change. You can now import using the aliases anywhere in your project:
```

How is something like this for getting across the idea that a restart is required, but is expected to happen automatically?

We try to avoid saying "should" in docs, so I'd prefer to say it "will" restart if that's actually the expected behavior. (Sure, it occasionally might not. But there are a lot of things that occasionally go wrong. 😄 )

But, this is a clue that if something isn't working, they can check to see whether the dev server actually *did* restart or do so manually.
